UPGRADE-2.8.md

```diff
@@ -475,6 +475,8 @@ Security
    }
    ```
 
+ * The `AbstractVoter` is deprecated. Upgrade to `Voter` instead.
+
```

Comment:
If we're deprecating `AbstractVoter` entirely, then we don't need to talk about it at all, right? The upgrade path would be to switch to `Voter` and make the changes needed there, correct? If so, I think we should have one simple, before and after (before `AbstractVoter` after with `Voter`).